vita3k/gui/src/initial_setup.cpp

```diff
@@ -210,6 +213,14 @@ void draw_initial_setup(GuiState &gui, EmuEnvState &emuenv) {
     case SELECT_INTERFACE_SETTINGS:
         title_str = lang["select_interface_settings"];
         ImGui::SetCursorPosY((WINDOW_SIZE.y / 2.f) - ImGui::GetFontSize());
+        ImGui::Checkbox("Display Info Message", &emuenv.cfg.display_info_message);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Uncheck the box to display info message in log only.");
```

Comment:
```suggestion
            ImGui::SetTooltip("Check the box to display a notification every time the compatibility database is updated.");
```

Reply:
Same, code is just copy pasted from setting dialog, i no want change it, like i said, is message d'or any info, just for now is onky set for compat